server/conf/messages

```diff
@@ -123,6 +123,10 @@ button.apply=Apply
 button.applySr=Apply to {0}
 # The text on the button an applicant clicks to edit the application for a specific program.
 button.edit=Edit
+# The screen reader text for a button allowing an applicant to edit a submitted application for a given program.
+button.editSr=Edit submitted application to {0}
```

Comment:
It in fact starts a new application. I think the idea behind using "Edit" was that when they navigated back, their previous answers would still be populated and they would only "edit" what had changed. Under the hood though, it would in fact create a new application.